ml-notebook/packages.txt

```diff
@@ -209,31 +208,31 @@ icu==69.1
 idna==3.3
 imagecodecs==2022.2.22
 imageio==2.16.1
-importlib-metadata==4.11.2
-importlib_metadata==4.11.2
-importlib_resources==5.4.0
+importlib-metadata==4.11.3
+importlib_metadata==4.11.3
+importlib_resources==5.6.0
 iniconfig==1.1.1
 intake==0.6.5
 intake-esm==2021.8.17
 intake-geopandas==0.4.0
 intake-stac==0.3.0
 intake-xarray==0.6.0
-ipykernel==6.9.1
+ipykernel==6.12.0
 ipyleaflet==0.15.0
 ipyspin==0.1.5
-ipython==8.1.1
+ipython==8.2.0
 ipython_genutils==0.2.0
 ipytree==0.2.1
 ipyurl==0.1.2
-ipywidgets==7.6.5
+ipywidgets==7.7.0
 isodate==0.6.1
 jasper==2.0.33
-jax==0.3.0
+jax==0.3.4
 jaxlib==0.1.75
```

Comment:
You'll need to figure out why jaxlib was not bumped here. BTW, that warning has nothing to do with my PR, it is from jax/jaxlib versionsing upstream checking. The PR just ensure you don't get a bad mix when using the new packages. Sadly, we old ones can still be mixed, but that error should help you avoid a bad mix.

Reply:
Thanks for the tip! FWIW, this original environment spec doesn't pin any versions, just a vanilla jax install

https://github.com/pangeo-data/pangeo-docker-images/blob/1affb9fbbdebbcc1023d7ad71f7151684e360c0b/ml-notebook/environment.yml#L1-L9

`packages.txt` is generated by conda-lock. So to us it looks like we did "get a bad mix" from the conda solver.

Reply:
Thanks! In 6496e5e I tried adding `jaxlib` to this environment.yml.

Reply:
That did not work, we are still getting
```
jax==0.3.6
jaxlib==0.1.75
```

I guess the next step would be to pin a jaxlib version?